### PR TITLE
Don't show anything by default on the shopping screen #270

### LINKED ISSUE #270: Don't show anything by default on the shopping screen
### Steps to reproduce
- Select `Travel` from the navigation drawer
- Select `Hotel Booking`

It automatically searches for bags & show results.
screen shot 2018-07-14 at 8 47 44 pm

### Expected behaviour
- Don't search for anything by default
- Show this animation on the screen in the center: https://www.lottiefiles.com/373-streetby-test-loading
- Below this, write a text, `Want to shop something? Select the search icon on top`

### PR DESCRIPTION
**Pull request by syedjahangirpeeran**
Fix #270

### CODE REVIEW

**Comment by Swati4star** [comment]
@syedjahangirpeeran Add a screenshot of animation and text message

**Comment by syedjahangirpeeran** [comment]
screenshot_1531760735

**Comment by Swati4star** [comment]
Your PR fixes this also then https://github.com/project-travel-mate/Travel-Mate/issues/276?

File: Android/app/src/main/res/layout/activity_shopping_currentcity.xml [hunk 1]
```diff
@@ -13,6 +14,24 @@
...
+            app:lottie_loop="true" />
+
+        <TextView
+            android:id="@+id/text_view"
+            android:layout_width="200dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|center"
+            android:layout_marginBottom="5dp"
+            android:gravity="center"
+            android:text="Want to shop something? Select the search icon on top." />
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_shopping_currentcity.xml]: Add constant string to strings.xml file

File: Android/app/src/main/java/io/github/project_travel_mate/travel/ShoppingCurrentCityActivity.java [hunk 2]
```diff
@@ -214,4 +208,9 @@ public static Intent getStartIntent(Context context) {
         Intent intent = new Intent(context, ShoppingCurrentCityActivity.class);
         return intent;
     }
+
+    private void networkError() {
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/travel/ShoppingCurrentCityActivity.java]: Add a small comment to this function

File: Android/app/src/main/java/io/github/project_travel_mate/travel/ShoppingCurrentCityActivity.java [hunk 3]
```diff
@@ -20,6 +20,7 @@
 import android.widget.EditText;
 import android.widget.ListView;
 import android.widget.ProgressBar;
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/travel/ShoppingCurrentCityActivity.java]: This import is no longer needed

**Comment by Swati4star** [comment]
Actually, #276 describes this only. That if we make a query with no internet, it keeps on showing the loader. So, your PR closes that issue also.

**Pull request merged**